package.json

```diff
@@ -82,6 +82,7 @@
     "@sentry/integrations": "^5.11.1",
     "@zxing/library": "^0.8.0",
     "abortcontroller-polyfill": "^1.4.0",
+    "analytics-node": "^3.4.0-beta.2",
```

Comment:
Any reason we're using a beta release, rather than the latest production version (`v3.3.0`)?